network-api/networkapi/buyersguide/pagemodels/product_update.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+class Update(models.Model):
+    source = models.URLField(
+        max_length=2048,
+        help_text='Link to source',
+        blank=True,
+    )
+
+    title = models.CharField(
+        max_length=256,
+        blank=True,
```

Comment:
This model doesn't have any required fields. Can we make `title` a required field?